### ISSUE TITLE
MavenCentral username and password properties no longer work

### ISSUE THREAD

**Issue by eygraber** [vanniktech/gradle-maven-publish-plugin#229]
The README says to use `mavenCentralRepositoryUsername` and `mavenCentralRepositoryPassword` but that fails when running the publish task. It says that `mavenCentralUsername` and `mavenCentralPassword` are missing. Renaming the properties to drop `Repository` works.

**Comment by gabrielittner** [comment]
Which Gradle version are you using?

**Comment by eygraber** [comment]
6.8.3 and 7.0

**Comment by gabrielittner** [comment]
Looks like I've read something the wrong way in the Gradle docs and now we're expecting the wrong property names for `closeAndReleaseRepository`. I'll fix that in a 0.15.1 release this week, then `mavenCentralUsername` and `mavenCentralPassword` will be used for everything. Thanks for reporting it

**Comment by eygraber** [comment]
I got this error now when running `closeAndReleaseRepository`. Is that related to this issue?

```
> Task :closeAndReleaseRepository FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Some problems were found with the configuration of task ':closeAndReleaseRepository' (type 'CloseAndReleaseRepositoryTask').
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.repositoryPassword' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'nexusOptions.repositoryPassword'.
      2. Mark property 'nexusOptions.repositoryPassword' as optional.
    
1 actionable task: 1 executed
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.repositoryUsername' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'nexusOptions.repositoryUsername'.
      2. Mark property 'nexusOptions.repositoryUsername' as optional.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.stagingProfile' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'nexusOptions.stagingProfile'.
      2. Mark property 'nexusOptions.stagingProfile' as optional.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
```

**Comment by eygraber** [comment]
I guess it is, because adding `mavenCentralRepositoryUsername` and `mavenCentralRepositoryPassword` gives me a different error:

```
> Task :closeAndReleaseRepository FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':closeAndReleaseRepository' (type 'CloseAndReleaseRepositoryTask').
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.stagingProfile' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'nexusOptions.stagingProfile'.
      2. Mark property 'nexusOptions.stagingProfile' as optional.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
```

**Comment by gabrielittner** [comment]
@eygraber A snapshot with your fix is up, so you could try it out. My general goal is to publish 0.15.1 this weekend

**Comment by gabrielittner** [comment]
0.15.1 with the fix is out

**Comment by eygraber** [comment]
Works great, thanks!

**Comment by gmale** [comment]
For what it's worth, in case it helps the next person, I just landed here after experiencing the same problem that manifested slightly differently, with this error:

```
Some problems were found with the configuration of task ':closeAndReleaseRepository' (type 'CloseAndReleaseRepositoryTask').
> No value has been specified for property 'nexusOptions.repositoryPassword'.
> No value has been specified for property 'nexusOptions.repositoryUsername'.
> No value has been specified for property 'nexusOptions.stagingProfile'.
```
First, I tried adding those values to `~/.gradle/gradle.properties` and that, of course, didn't work. So I landed here, then simply changed `0.15.0` to `0.15.1`.  Did nothing else but re-run `./gradlew closeAndReleaseRepository` and it worked perfectly. Waited for close and everything. It was beautiful.

This is a much better user flow. Thanks so much for creating and fixing this feature :clap:
